src/kem/kyber768/ntt.rs

```diff
@@ -41,21 +42,21 @@ pub(crate) mod kyber_polynomial_ring_element_mod {
                 zeta_i += 1;
 
                 for j in offset..offset + layer {
-                    let t = re[j + layer] * ZETAS[zeta_i];
-                    re[j + layer] = re[j] - t;
-                    re[j] = re[j] + t;
+                    let t = fe_mul(re[j + layer], ZETAS[zeta_i]);
```

Comment:
For readability, we might want to implement the operators. But let's keep this for now.